astro/src/content/docs/extend/code/lambdas/mfa-requirement.mdx

```diff
@@ -0,0 +1,315 @@
+---
+title: MFA Requirement Lambda
+description: An overview of the MFA requirement lambda.
+navcategory: customization
+section: extend
+subcategory: code
+tertcategory: lambdas
+---
+import AuthenticationTypeValues from 'src/content/docs/_shared/authentication-type-values.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import EnterprisePlanBlurb from 'src/content/docs/_shared/_enterprise-plan-blurb.astro';
+import InlineField from 'src/components/InlineField.astro';
+
+<EnterprisePlanBlurb />
+
+During logins, password changes, and [MFA Status API](/docs/apis/two-factor#retrieve-multi-factor-status) calls, FusionAuth performs various validations to decide whether to challenge the user on one of their MFA methods. Using this lambda function, you may extend this functionality to change FusionAuth's MFA decision for that user and/or trigger a suspicious login event.
```

Comment:
```suggestion
During logins, password changes, and [MFA Status API](/docs/apis/two-factor#retrieve-multi-factor-status) calls, FusionAuth performs various validations to decide whether to challenge the user on one of their MFA methods. Using this lambda function, you may extend this functionality to change FusionAuth's MFA decision for that user or trigger a suspicious login event.
```
[suggestion]: prefer or to and/or unless the 'and' contributes meaningful semantics